content/en/docs/setup/platform-setup/ibm/index.md

```diff
@@ -37,7 +37,7 @@ To prepare a cluster before manually installing Istio, proceed as follows:
     {{< /tip >}}
 
     {{< text bash >}}
-    $ ibmcloud ks cluster-create --zone <zone-name> --machine-type b3c.4x16 \
+    $ ibmcloud ks cluster create classic --zone <zone-name> --machine-type b3c.4x16 \
```

Comment:
This command doesn't seem to work for me.
```
$ ibmcloud ks cluster create classic --zone tor01 --machine-type b3c.4x16   --workers 3 --name mycluster
Creating cluster...
FAILED
One or more required fields are missing. To create a standard cluster, you must specify values for name, zone, machine type (flavor), public VLAN, and private VLAN. If this is the first standard cluster that you create in this zone, the public and private VLAN flags are not required. To create a free cluster, specify a cluster name only. (E0020)
```

Reply:
@frankbu Did you check the note and verify you don't have any vlans in that zone, else the vlans are required. Maybe we should change the `can specify` to `must specify`.